Source/PTEDashboard.m

```diff
@@ -59,9 +59,9 @@ - (instancetype)initWithFrame:(CGRect)frame
         }
         
         // Load Storyboard
-        self.rootViewController = [[UIStoryboard storyboardWithName:@"LumberjackConsole"
-                                                             bundle:nil] instantiateInitialViewController];
-        
+			  UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"LumberjackConsole" bundle:[NSBundle bundleForClass:[self class]]];
+			  self.rootViewController = [storyboard instantiateInitialViewController];
+			
```

Comment:
Should be ok now, the problem is that it is perfectly indented in my Xcode. 
Xcode automatically align the indentation